server/services/historicalPrisonerService.ts

```diff
@@ -1,33 +1,42 @@
-import HistoricalPrisonerApiClient from '../data/historicalPrisonerApiClient'
-import { PagedModelPrisoner } from '../@types/historical-prisoner/historicalPrisonerApiTypes'
+import querystring from 'querystring'
+import {
+  FindPrisonersByIdentifiers,
+  FindPrisonersByName,
+  PagedModelPrisonerSearchDto,
+} from '../@types/historical-prisoner/historicalPrisonerApiTypes'
+import RestClient from '../data/restClient'
+import config from '../config'
 
 export default class HistoricalPrisonerService {
-  constructor(private readonly historicalPrisonerApiClient: HistoricalPrisonerApiClient) {}
+  constructor() {}
+
+  private static restClient(token: string): RestClient {
+    return new RestClient('Historical Prisoner Api Client', config.apis.historicalPrisonerApi, token)
+  }
 
   async findPrisonersByName(
     token: string,
-
-    nameSearchForm: {
-      forename?: string
-      surname?: string
-      dateOfBirth?: string
-      page?: number
-    },
-  ): Promise<PagedModelPrisoner> {
-    return this.historicalPrisonerApiClient.findPrisonersByNameOrAge(token, nameSearchForm)
+    prisonersByNameForm: FindPrisonersByName,
+  ): Promise<PagedModelPrisonerSearchDto> {
```

Comment:
Have removed the apiclient - not sure if we need both service and client calls as a lot of the work is actually done in the controller.  Service was just acting like the apiclient